### PR TITLE
Move to new project system

### PR DESCRIPTION
**Pull request by slang25**
This is in preparation of porting to netstandard1.x

Dependencies are close very close to being ported

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,24 +1,18 @@
 version: 1.0.0.{build}
+image: Visual Studio 2017 Preview
```
Comment (martincostello) [inline appveyor.yml]: How come preview?

File: src/JustBehave.Common.props [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <Authors>Just Eat</Authors>
+    <Company>Just Eat</Company>
+    <Copyright>Copyright 2011-$([System.DateTime]::Now.ToString(yyyy)) Just Eat, Inc. or its affiliates. All Rights Reserved.</Copyright>
+    <Description>A BDD-influenced C# testing library cooked up by JUST EAT</Description>
```
Comment (martincostello) [inline src/JustBehave.Common.props]: The pedant in me says this should be Title Case not all capitals now.

File: appveyor.yml (hunk 1 above)
Comment (slang25) [inline appveyor.yml]: Literally because I felt like it 😆 I'll change this back

File: src/JustBehave.Common.props [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    <Version>$(APPVEYOR_BUILD_VERSION)</Version>
+    <Version Condition=" '$(Version)' == '' ">0.1.0</Version>
```
Comment (martincostello) [inline src/JustBehave.Common.props]: Should this be `1.0.0`? The AppVeyor builds seem to be `1.0.x.x`.

**Comment by martincostello** [comment]
👍

**Comment by AnthonySteele** [comment]
Looks good to me. And it builds.

**Comment by slang25** [comment]
This one is ready to merge as well if you care to do the honors  @AnthonySteele

**Pull request merged**